src/main/java/com/sparta/popupstore/domain/review/controller/ReviewController.java

```diff
@@ -1,12 +1,30 @@
 package com.sparta.popupstore.domain.review.controller;
 
+import com.sparta.popupstore.domain.review.dto.request.ReviewSaveRequestDto;
+import com.sparta.popupstore.domain.review.dto.response.ReviewSaveResponseDto;
 import com.sparta.popupstore.domain.review.service.ReviewService;
+import com.sparta.popupstore.domain.user.entity.User;
+import com.sparta.popupstore.domain.user.repository.UserRepository;
 import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 @RequiredArgsConstructor
 public class ReviewController {
 
     private final ReviewService reviewService;
+    private final UserRepository userRepository;
+
+    @PostMapping("/reviews/popupstores/{id}")
+    public ResponseEntity<ReviewSaveResponseDto> createReview(@PathVariable Long id, @RequestBody ReviewSaveRequestDto requestDto) {
```

Comment:
```suggestion
    @PostMapping("/reviews/popupstores/{popupstoreId}")
    public ResponseEntity<ReviewSaveResponseDto> createReview(@PathVariable Long popupstoreId, @RequestBody ReviewCreateRequestDto requestDto) {
```
id 보다는 명확하게 명시하는게 좋습니다.
그리고 Save는 Create로 수정해주세요.